<launch>

  <node pkg="minimumsnap" type="trajectory_generator_node" name="trajectory_generator_node" output="screen" required = "true">
    <remap from="~odom"      to="odometryTopic"/>

    <!-- 规划器参数 -->
    <param name="planning/vel"       value="2.0"  />
    <param name="planning/acc"       value="2.0"  />
    <param name="planning/dev_order" value="4"    />
    <param name="planning/min_order" value="3"    />
    <param name="vis/vis_traj_width" value="0.15" />

    <!-- 参考柱子坐标 -->
    <param name="planning/pillar1_x" value="3" />
    <param name="planning/pillar1_y" value="0" />
    <param name="planning/pillar1_z" value="1.5" />

    <param name="planning/pillar2_x" value="5" />
    <param name="planning/pillar2_y" value="0" />
    <param name="planning/pillar2_z" value="1.5" />

    <param name="time_forward" value="1.0" type="double" />

  </node>

  <!-- <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find trajectory_generator)/rviz_config/nuc.rviz" /> -->

</launch>
